luarules/gadgets/game_team_power_watcher.lua

```diff
@@ -0,0 +1,294 @@
+
+
+function gadget:GetInfo()
+	return {
+		name = "Team Power Watcher",
+		desc = "Tracks power of individual and total units per team. To be used for PvE dynamic difficulty and library functions to assertain aspects of game progression",
+		author = "SethDGamre",
+		date = "2024-07-12",
+		license = "None",
```

Comment:
Remove this line

Reply:
I don't understand why, but I'll remove these 4 lines blindly because you asked.

Reply:
Just the license line, that is the line the comment is on

Reply:
Oh, ok